OpenTabletDriver.Configurations/Parsers/Wacom/BambooPad/BambooPadAuxReport.cs

```diff
@@ -0,0 +1,23 @@
+using System.Numerics;
+using System.Runtime.CompilerServices;
+using OpenTabletDriver.Tablet;
+
+namespace OpenTabletDriver.Configurations.Parsers.Wacom.BambooPad
+{
+    public struct BambooPadAuxReport : IAuxReport
+    {
+        public BambooPadAuxReport(byte[] report)
+        {
+            Raw = report;
+
+            AuxButtons = new bool[]
+            {
+                report[23] == 1,
+                report[23] == 2

```

Comment:
This prevents it from sending both buttons as active. Is it not possible for the hardware to send both?

```suggestion
                report[23].IsBitSet(0),
                report[23].IsBitSet(1)
```

Reply:
As far as i tested, This value is only returns 1 or 2, It is not possible to click both buttons as same time even on official drivers. It is also prioritize right button.

Reply:
After another test i found problem related to long press, maybe i will try improve it later.